server/routes/index.js

```diff
@@ -1,8 +1,15 @@
 const express = require('express');
+const path = require('path');
+
 const apiRouter = require('./api');
 
 const router = express.Router();
 
 router.use('/api', apiRouter);
 
+router.get('*', (req, res) => {
+  res.sendFile(path.join(__dirname, '../../browser/dist/index.html'));
+});
```

Comment:
I added back `router.get('*', (req, res) => {}` which I erroneously removed while getting rid html template rendering in a previous commit. I also added `res.sendFile(path.join(__dirname, '../../browser/dist/index.html'));` within the GET in order to direct requests to `index.html` for client-side routing.

Reply:
Wouldn't this give you an error (index.html file doesn't exist), if `webpack-dev-server` is loaded? My understanding is that we will not have an `index.html` file when we're working on `dev`. I see you still have the WIP label on this. Apologies if you're still working on this.

Reply:
Yeah I get that error when entering a url with server port : 
```Error: ENOENT: no such file or directory, stat '/Users/benjaminsinaiko/projects/site-builder/browser/dist/index.html'```
Think this also relates to our question regarding best way to handle routing/rerouting with reset password email link.

Reply:
I'm getting the same that Ben got.

Reply:
I believe I fixed your problem. Give it another the shot.